projects/power_distribution/src/main.c

```diff
@@ -1,6 +1,14 @@
 // Uncomment this line to force firmware to run as front or rear power distribution.
 // #define FORCE_IS_FRONT_POWER_DISTRIBUTION true
 
+// SMOKE TEST MODES: each macro is a different smoke test you can run.
+
+#define SMOKE_TEST_MODE_NORMAL 0               // run as normal PD
```

Comment:
Calling normal operation a smoke test mode makes it a bit harder to understand. Maybe `OP_MODE_NORMAL` vs `OP_MODE_SMOKE_CURRENT_MEASUREMENT` or something like that?